src/main/java/com/example/app/util/Utils.java

```diff
@@ -0,0 +1,12 @@
+package com.example.app.util;
+
+/**
+ * Class that contains all service constants and utils for the application.
```

Comment:
What about storing constants in .property file?

Reply:
I don't think it's a good idea because then we'll need additional resources for extracting that data from the .property file.
Perhaps we'll think about it sometime, if there are too many constants.